### PR TITLE
fix(iOS): Fire the native SizeChanged event upon init of Window

### PR DESCRIPTION
**Pull request by kazo0**
fixes https://github.com/unoplatform/nventive-private/issues/402

## PR Type

What kind of change does this PR introduce?
- Bugfix

## What is the current behavior?

Attempting to retrieve the Window.Bounds too early results in a Rect of 0x0

## What is the new behavior?

Initialization of the Windows.UI.XamlWindow fires a SizeChanged event to ensure applications will not recieve an empty Rect when trying to access the Window.Bounds

### CODE REVIEW

File: src/Uno.UI/Extensions/ViewHelper.iOSmacOS.cs [hunk 1]
```diff
@@ -231,11 +231,12 @@ public static nfloat StackSubViews(_View thisView, float topPadding, float space
 		/// Gets the orientation-dependent screen size
 		/// </summary>
 		/// <returns></returns>
-		public static CGSize GetScreenSize()
+		public static CGSize GetScreenSize(Window window = null)
 		{
 #if __IOS__
-			var width = Window.Current.NativeWindow.Frame.Width;
-			var height = Window.Current.NativeWindow.Frame.Height;
+			var nativeWindow = window?.NativeWindow ?? Window.Current.NativeWindow;
```
Comment (jeromelaban) [inline src/Uno.UI/Extensions/ViewHelper.iOSmacOS.cs:237]: We should be validating if `Window.Current.NativeWindow` is null as well.

**Review by jeromelaban — APPROVED** [review]

**Review by MartinZikmund — APPROVED** [review]

**Comment by MartinZikmund** [comment]
One potential issue is that this is a breaking change:

```
Removed method CoreGraphics.CGSize Uno.UI.ViewHelper.GetScreenSize() not found in ignore set.
```

It may be possible that this is used by some nventive libraries? @jeromelaban In such case we could create a separate `internal` overload instead and call it from the parameterless version with `Window.Current` as parameter.

**Comment by kazo0** [comment]
> One potential issue is that this is a breaking change:
> 
> ```
> Removed method CoreGraphics.CGSize Uno.UI.ViewHelper.GetScreenSize() not found in ignore set.
> ```
> 
> It may be possible that this is used by some nventive libraries? @jeromelaban In such case we could create a separate `internal` overload instead and call it from the parameterless version with `Window.Current` as parameter.

Ah damn I didn't realize the ViewHelper was public

**Comment by MartinZikmund** [comment]
@kazo0 Surprised me too when I was changing it 😀

**Review by spouliot — APPROVED** [review]

**Pull request merged**